java/src/org/openqa/selenium/grid/commands/EventBusCommand.java

```diff
@@ -24,12 +24,7 @@
 import static org.openqa.selenium.remote.http.Contents.asJson;
 
 import com.google.auto.service.AutoService;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Can you use single imports, please?

Reply:
Ahh, apologies, laptop re-formatted and missed adding checkstyle. Will fix it